linux/task.go

```diff
@@ -253,6 +255,9 @@ func (t *Task) Process(ctx context.Context, id string) (runtime.Process, error)
 
 // Metrics returns runtime specific system level metric information for the task
 func (t *Task) Metrics(ctx context.Context) (interface{}, error) {
+	if t.cg == nil {
+		return nil, errors.Wrap(errdefs.ErrNotFound, "cgroup does not exist")
+	}
 	stats, err := t.cg.Stat(cgroups.IgnoreNotExist)
```

Comment:
Can we not `IgnoreNotExist` error here, and return `ErrNotFound` error?
@crosbymichael

Reply:
This is hard because it will error if one subsystem does not exit.  We would have to modify the cgroup code more to see if none of the cgroups exits.

Reply:
I see. Then that's fine.